setup.py

```diff
@@ -14,13 +14,16 @@
 
 import glob
 import os
+import platform
 import shutil
 import subprocess
 import sys
 import zipfile
 from pathlib import Path
+from typing import List
 
 from setuptools import find_packages, setup
+from typing_extensions import TypedDict
```

Comment:
This would break the build when this isn't installed.

Reply:
reverted